addons/index.md

```diff
@@ -7,16 +7,108 @@ layout: documentation
+| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items                               |
+| [Voice Services](voices.html)           | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc.                                |
+| [3rd Party System Integration](io.html) | Expose openHAB to external systems                                                                                        |
+
+## Installation of Add-ons
+
+Depending on the [package]({{base}}/configuration/packages.html) you have choosen during your first time setup, there are already some pre-installed add-ons.
+Additional add-ons can be installed in the different ways, described below.
+
+### Through Paper UI
+
+Navigate to the add-ons section.
+Search for the desired add-on in the categories and press install.
+
+![installing Add-ons through Paper UI](images/index_installation_paperui.gif)
+
+### By editing addons.cfg
+
+For this installation option you need to know the *id* of the desired add-on
+
+You can find it out in the [openHAB console]({{base}}/administration/console.html) with:
+
+```sh
+feature:list | grep ^openhab
+```
+
+This will give you a list of all available add-ons.
+It could probably look similar to this example:
+
+```text
+...
+openhab-transformation-xslt               | 0.9.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | XSLT Transformation
+openhab-voice-mactts                      | 0.9.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | MacOS Text-to-Speech
+openhab-binding-amazondashbutton          | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Amazon Dash Button Binding
+openhab-binding-astro                     | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Astro Binding
+openhab-binding-autelis                   | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Autelis Binding
+openhab-binding-avmfritz                  | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | AVM Fritz!Box Binding
+...
+openhab-binding-network                   │ 2.2.0            │          │ Uninstalled │ openhab-addons-2.2.0    │ Network Binding
+...
+```
+
+According to the [naming convention for bundles]({{base}}/administration/bundles.html#naming-convention-for-bundles) the *id* for our example is *network*.
+
+Another way of getting the correct id, is the url from the add-ons documentation page.
+For example the url for the [mqtt binding]({{base}}/addons/bindings/mqtt1/readme.html) is
+
+````text
+https://docs.openhab.org/addons/bindings/mqtt1/readme.html
+```
+
+So the id would be *mqtt1* in this case.
+Did you notice the trailing *1* in this id?
+This is because the mqqt binding is a 1.x add-on.
+
+> All 1.x add-ons **must** have a trailing 1 in their id.
```

Comment:
Must suggests the user has to take care of this. However following both methods above you'll end up with the 1 automatically. 'd remove this sentence